builds/android/app/src/main/res/values-fr/strings.xml

```diff
@@ -24,36 +23,53 @@
   <string name="no">Non</string>
   <string name="creating_dir_failed">La création du dossier $PATH a échoué</string>
   <string name="path_not_readable">$PATH non lisible</string>
+  <string name="no_games_found_and_explanation">Pas de jeu RPG Maker 2000/2003 trouvé.
+Vous pouvez ajouter des dossiers de jeux dans les paramètres.</string>
   <string name="no_external_storage">Pas de stockage externe trouvé</string>
   <string name="not_valid_game">$PATH n\'est pas un jeu valide</string>
   <string name="select_game_region">Changer la région du jeu</string>
   <string name="change_the_layout">Changer les contrôles virtuels</string>
-  <string name="choose_layout">Choisir un schéma de contrôles virtuels</string>
+  <string name="choose_layout">Choisir les contrôles virtuels</string>
   <string name="unknown_region">Région inconnue</string>
   <string name="region_modification_failed">Le changement de la région a échoué</string>
   <string name="refresh">Raffraichir</string>
   <string name="change_default_mapping">Changer les boutons virtuels</string>
   <string name="how_to_use_easy_rpg">Aide</string>
-  <string name="how_to_use_easy_rpg_explanation">
-Instructions :\n
-\t1. Placer les dossiers de jeux dans le dossier /easyrpg/games créé au lancement de l\'application.\n
-\t2. Placer les dossiers de RTP de RPG Maker 2000 et 2003 dans les dossiers /easyrpg/rtp/2000 et /easyrpg/rtp/2003.\n
-\n
-Si vous ne trouvez pas le dossier /easyrpg, alors celui-ci se trouve sur un autre support (carte SD ou mémoire interne).\n
-\n
-Les dossiers de jeux doivent être décompressés et non placés dans un sous dossier afin d\'être détéctés.</string>
+  <string name="how_to_use_easy_rpg_explanation">Instruction :\n
+Par défaut les jeux sont scannés dans le dossier /easyrpg/games de l\'application.
+Vous pouvez définir de nouveaux dossiers de jeux via les paramètres.
+
+Optionnel :\n
+Placez les RTP dans les dossier /easyrpg/rtp/2000 et /easyrpg/rtp/2003.</string>
```

Comment:
Arf I thought Transifex would do this for me (it displays an "enter" icon in preview), apologies. Fixed.

For the translation of the path, it seems that you're reffering to the old translation. Besides, the path is not /easyrpg/game but, well, android.